ScriptFileWatcherTest unstable

This test failed in: https://github.com/openhab/openhab-core/runs/5605602828

```
[main] INFO org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher - Loading script '/tmp/junit290363953962692893/script.js'
[ERROR] Tests run: 11, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 2.577 s <<< FAILURE! - in org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcherTest
[ERROR] org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcherTest.testLoadOneDefaultFileDelayedSupport  Time elapsed: 0.221 s  <<< FAILURE!
org.mockito.exceptions.verification.WantedButNotInvoked: 

Wanted but not invoked:
scriptEngineManager.createScriptEngine(
    "js",
    "file:/tmp/junit4986597073130184316/script.js"
);
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcherTest.testLoadOneDefaultFileDelayedSupport(ScriptFileWatcherTest.java:239)

However, there were exactly 3 interactions with this mock:
scriptEngineManager.addFactoryChangeListener(
    org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher@19868320
);
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher.<init>(ScriptFileWatcher.java:90)

scriptEngineManager.isSupported("js");
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher.lambda$2(ScriptFileWatcher.java:222)

scriptEngineManager.isSupported("js");
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher.lambda$5(ScriptFileWatcher.java:287)


	at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcherTest.testLoadOneDefaultFileDelayedSupport(ScriptFileWatcherTest.java:239)

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   ScriptFileWatcherTest.testLoadOneDefaultFileDelayedSupport:239 
Wanted but not invoked:
scriptEngineManager.createScriptEngine(
    "js",
    "file:/tmp/junit4986597073130184316/script.js"
);
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcherTest.testLoadOneDefaultFileDelayedSupport(ScriptFileWatcherTest.java:239)

However, there were exactly 3 interactions with this mock:
scriptEngineManager.addFactoryChangeListener(
    org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher@19868320
);
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher.<init>(ScriptFileWatcher.java:90)

scriptEngineManager.isSupported("js");
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher.lambda$2(ScriptFileWatcher.java:222)

scriptEngineManager.isSupported("js");
-> at org.openhab.core.automation.module.script.rulesupport.loader.ScriptFileWatcher.lambda$5(ScriptFileWatcher.java:287)


[INFO] 
[ERROR] Tests run: 11, Failures: 1, Errors: 0, Skipped: 0
```

--- a/bundles/org.openhab.core.automation.module.script.rulesupport/src/test/java/org/openhab/core/automation/module/script/rulesupport/loader/ScriptFileWatcherTest.java
+++ b/bundles/org.openhab.core.automation.module.script.rulesupport/src/test/java/org/openhab/core/automation/module/script/rulesupport/loader/ScriptFileWatcherTest.java
@@ -107,7 +107,7 @@ class ScriptFileWatcherTest {
 
         scriptFileWatcher.processWatchEvent(null, ENTRY_CREATE, p);
 
-        verify(scriptEngineManager).createScriptEngine("js", p.toFile().toURI().toString());
+        verify(scriptEngineManager, timeout(1000)).createScriptEngine("js", p.toFile().toURI().toString());
     }
 
     @Test
@@ -236,7 +236,7 @@ class ScriptFileWatcherTest {
         scheduledTask.getValue().run();
 
         // verify script has now been processed
-        verify(scriptEngineManager, times(1)).createScriptEngine("js", p.toFile().toURI().toString());
+        verify(scriptEngineManager, timeout(1000).times(1)).createScriptEngine("js", p.toFile().toURI().toString());
     }
 
     @Test
@@ -281,7 +281,7 @@ class ScriptFileWatcherTest {
 
         scriptFileWatcher.onDependencyChange(p.toFile().toURI().toString());
 
-        verify(scriptEngineManager, times(2)).createScriptEngine("js", p.toFile().toURI().toString());
+        verify(scriptEngineManager, timeout(1000).times(2)).createScriptEngine("js", p.toFile().toURI().toString());
     }
 
     @Test
@@ -344,6 +344,6 @@ class ScriptFileWatcherTest {
         scriptFileWatcher.processWatchEvent(null, ENTRY_MODIFY, p);
 
         verify(scriptEngineManager).removeEngine(p.toFile().toURI().toString());
-        verify(scriptEngineManager, times(2)).createScriptEngine("js", p.toFile().toURI().toString());
+        verify(scriptEngineManager, timeout(1000).times(2)).createScriptEngine("js", p.toFile().toURI().toString());
     }
 }

